packages/wallets/src/features/cashier/modules/TransactionStatus/components/TransactionStatusError/__tests__/TransactionStatusError.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import TransactionStatusError from '../TransactionStatusError';
+
+describe('TransactionStatusError', () => {
+    it('should render component correctly', () => {
+        const mockRefresh = jest.fn();
+
+        render(<TransactionStatusError refresh={mockRefresh} />);
+
+        expect(screen.getByText('Unfortunately, we cannot retrieve the information at this time.')).toBeInTheDocument();
+
+        const refreshButton = screen.getByText('Refresh');
+        expect(refreshButton).toBeInTheDocument();
+
+        fireEvent.click(refreshButton);
```

Comment:
for future PRs: probably better to follow one given/when/then for one test case, 
this test case is named "should render component correclty" but its also testing its logic, so that probably separate given/when/then and separate unit tests (its unit test, so its really cheap to run, no need to aggregate them together in order to save runs)

Reply:
Will keep that in mind for future PRs 👍 Updated the unit test to separate the test cases